Playground/js/main.js

```diff
@@ -40,7 +40,7 @@ compileAndRun = function(parent, fpsLabel) {
         var createEngineFunction = "createDefaultEngine";
         var createSceneFunction;
 
-        parent.monacoCreator.getRunCode(function (code) {
+        parent.monacoCreator.getRunCode().then(function (code) {

```

Comment:
I recommend using () => instead of function()

Reply:
Fully agree. Given I'm a total newbie in webdev, from my understanding with Arrow functions you don't have to create "this closures" or even use bind(this) like it was done in those functions right ?

Reply:
lol if all newbies were like you :D

yes it capture the this for you and stay in the "logical" context. No more that= this!!